src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,30 @@
+package duke.tasks;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Event extends Task {
+
+    protected Date from;
+    protected Date to;
+    protected String fromFormat;
+    protected String toFormat;
```

Comment:
Good job separating the names between Date and String and having similar naming formats, might be better to change the name of the variable here as it does not seem to be clear as to what format this variable is.
```suggestion
    protected String fromStringFormat;
    protected String toStringFormat;
```